src/main/flight/pid.h

```diff
@@ -82,6 +82,7 @@ typedef struct pidProfile_s {
     uint16_t pidSumLimit;
     uint16_t pidSumLimitYaw;
     uint8_t vbatPidCompensation;            // Scale PIDsum to battery voltage
+    uint8_t vbatThrottleCompensation;       // Scale throttle to battery voltage
```

Comment:
This breaks PG. New value should do to end of struct (or version has to be incremented)